emulated_hue/discovery.py

```diff
@@ -42,7 +42,6 @@ def start_zeroconf_discovery(config: Config):
             "bridgeid": config.bridge_id,
             "modelid": config.definitions["bridge"]["basic"]["modelid"],
         },
-        server=f"hue_{config.bridge_id[-6:]}.local.",
```

Comment:
Does paring still work without this? I believe you added this earlier due to a paring issue.

Reply:
it is not present in the real bridge and pairing works without it

Reply:
Sounds good